iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/SSLContextBuilder.java

```diff
@@ -23,28 +22,31 @@
 import java.security.spec.PKCS8EncodedKeySpec;
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.UUID;
 
 public class SSLContextBuilder
```

Comment:
This is a copy of the SSLContextBuilder that lives in our samples that we use for our e2e tests. It isn't part of the samples that I'm editing, but it is the same code so it may as well be reviewed at the same time